contrib/flake.nix

```diff
@@ -142,6 +142,10 @@
                 jq # jq for scripts/vim-patch.sh -r
                 shellcheck # for `make shlint`
                 doxygen # for script/gen_vimdoc.py
+              ];
+
+            nativeBuildInputs = with pkgs;
+              oa.nativeBuildInputs ++ [
```

Comment:
while you are at it, could you remove cmake too ? It should be available in the nixpkgs derivation too